lib/Core/PTree.cpp

```diff
@@ -9,14 +9,21 @@
 
 #include "PTree.h"
 
+#include "klee/ExecutionState.h"
 #include "klee/Expr/Expr.h"
 #include "klee/Expr/ExprPPrinter.h"
+#include "klee/Internal/Module/KInstruction.h"
+#include "klee/Internal/Support/FileHandling.h"
```

Comment:
Are all this headers still neccessary?

Reply:
Removed FIleHandling.h, but still need KInstruction.h